docs/docs/background-jobs.md

```diff
@@ -0,0 +1,670 @@
+# Background Jobs
+
+No one likes waiting in line. This is especially true of your website: users don't want to wait for things to load that don't directly impact the task they're trying to accomplish. For example, sending a "welcome" email when a new user signs up. The process of sending the email could as long or longer than the sum total of everything else that happens during that request. Why make the user wait for it? As long as they eventually get the email, everything is good.
+
+A typical create-user flow could look something like this:
+
+![image](/img/background-jobs/jobs-before.png)
+
+If we want the email to be send asynchonously, we can shuttle that process off into a **background job**:
+
+![image](/img/background-jobs/jobs-after.png)
+
+The user's response is returned much quicker, and the email is sent by another process which is connected to a user's session. All of the logic around sending the email is packaged up as a **job** and a **job worker** is responsible for executing it.
```

Comment:
> which is connected to a user's session

Makes me think of auth sessions and cookies. Might just be me though.

Reply:
Yeah not sure why I included that, I just reworded it.